pennylane/templates/state_preparations/mottonen.py

```diff
@@ -26,7 +24,7 @@
 from pennylane.wires import Wires
 
 
-# pylint: disable=len-as-condition,arguments-out-of-order
+# pylint: disable=len-as-condition,arguments-out-of-order,consider-using-enumerate
```

Comment:
I am avoiding enumerate because I suspect that it is faster to index a tensor than to enumerate over it.